include/inria_wbc/utils/robot_model.hpp

```diff
@@ -60,6 +60,20 @@ namespace utils {
         pinocchio::Data::Matrix6x jacobian(const std::string& joint_name, const pinocchio::ReferenceFrame& reference_frame = pinocchio::WORLD);
         pinocchio::Data::Tensor3x hessian(const std::string& joint_name, const pinocchio::ReferenceFrame& reference_frame = pinocchio::WORLD);
 
+        //Compute torques from sensor_data external forces.
+        //It is here done for the two foot contacts but it could be generalized to less or more contacts if needed
+        //You could directly use pinocchio rnea if there is no external forces or no foot mass to add
+        void compute_rnea_double_support(const std::unordered_map<std::string, Eigen::MatrixXd>& sensor_data,
+            const Eigen::VectorXd& q, 
+            const Eigen::VectorXd& v,
+            const Eigen::VectorXd& a, 
+            Eigen::VectorXd& tau_model,
+            bool add_foot_mass = true,
+            const std::string& left_ft_frame = "leg_left_6_joint",
```

Comment:
I know these are only default values for arguments, but this seems to be very talos-specific. I would vote to remove the defaut values.

Reply:
yes I have removed the default values. It is still humanoid specific and double support specific. I think it could be improved by making the number of contacts and associated sensor_data external forces generic. It could be done in another PR. The main change with pinocchio default rnea functions is the possibility to deal with the mass of the body between the F/T sensor and the contact.